src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/AbstractJavadocCheck.java

```diff
@@ -351,4 +355,16 @@ private boolean shouldBeProcessed(DetailNode curNode) {
         return javadocTokens.contains(curNode.getType());
     }
 
+    /**
+     * This method determines if a check should process javadoc containing unclosed html tags.
+     * This method must be overridden in checks extending {@code AbstractJavadocCheck} which
+     * are not supposed to process javadoc containing unclosed html tags.
+     * @return true if the check should or can process javadoc containing unclosed html tags;
+     *     false otherwise
+     * @see JavadocUtils#hasUnclosedTag(DetailNode)
+     */
+    public boolean doesCheckAcceptJavadocWithUnclosedHtmlTag() {
```

Comment:
Added this method which can be overriden by extending checks rather than using a field since this looked better. 
If a field was used then either it would have to be overwritten using constructors in extending subclasses or using instance initialization blocks. Also, there would have been problem making fields public and then we would have required a public _getter_ kind of method.